SELF-ASSESMENT.md

```diff
@@ -17,4 +17,42 @@ This is a placeholder document for the Jaeger project self-assessment. More deta
 * [Project compliance](#project-compliance)
 * [Secure development practices](#secure-development-practices)
 * [Security issue resolution](#security-issue-resolution)
-* [Appendix](#appendix)## Table of Contents
\ No newline at end of file
+* [Appendix](#appendix)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/jaegertracing/jaeger/  |
+| Security Provider | No  |
+| Languages | Go |
+| SBOM | [Software bill of materials](https://github.com/jaegertracing/jaeger/releases/latest/download/jaeger-SBOM.spdx.json) |
+| | |
+
+### Security links
+
+Provide the list of links to existing security documentation for the project. You may
+use the table below as an example:
+| Doc | url |
+| -- | -- |
+| Security file | https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md |
+
+## Overview
```

Comment:
Can we drop this and link to the website or readme? I don't want to maintain these duplicate descriptions

Reply:
The specification requires that this is in a standalone file. I did not find docs on the website for security at all, they are only in the repo.

Reply:
I am not saying not to have this file, but just remove the content of the Overview section and write "See README", so that we don't repeat text.

Reply:
That needs to be there per the spec, it's kind of redundant, but I believe they are parsing it which is why we aren't passing :) https://github.com/ossf/security-insights-spec/blob/v1.0.0/specification.md